spec/rubocop/cop/performance/use_zip_to_wrap_array_contents_spec.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::UseZipToWrapArrayContents, :config do
+  context 'when using map with array literal' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        [1, 2, 3].map { |id| [id] }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `.zip` instead of `.map { |id| [id] }`.
+      RUBY
+    end
+
+    it 'corrects the code' do
+      new_source = autocorrect_source('[1, 2, 3].map { |id| [id] }')
+
+      expect(new_source).to eq '[1, 2, 3].zip'
+    end
+  end
+
+  context 'when using map with a short iterator name' do
+    let(:source) do
+      <<~RUBY
+        [1, 2, 3].map { |e| [e] }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^ Use `.zip` instead of `.map { |id| [id] }`.
+      RUBY
+    end
+
+    it 'registers an offense' do
+      expect_offense(source)
```

Comment:
Use `expect_offense(<<~RUBY)` instead of putting it in `let`. Thats largely the convention  over the codebases. Same for other instances.

Reply:
Thanks, will do!

Reply:
And fixed!